Automattic/wp-calypso
# Use Correct Jetpack Business Plan Slug from Marketing Tools Gmb Link

#### Changes proposed in this Pull Request

* Use Jetpack Business Plan slug when viewing `/marketing/tools` on a Jetpack site
* Add Google My Business as an official Jetpack Business "Feature"
* Misc typing fixes

#### Testing instructions

1. Navigate to `/marketing/tools` with a site with Jetpack Premium or less
2. Verity the "Let your customers find you on Google" card matches the following screenshot:
<img width="496" alt="Screen Shot 2020-04-16 at 2 35 41 PM" src="https://user-images.githubusercontent.com/2810519/79509848-2f8f7c80-7ff1-11ea-9326-34c5c33caad0.png">
3. Click "Upgrade to Professional" and verify it takes you to `/plans` with Professional "Suggested"
<img width="200" alt="Screen Shot 2020-04-16 at 2 48 48 PM" src="https://user-images.githubusercontent.com/2810519/79509960-636aa200-7ff1-11ea-839a-36bdf34d105f.png">
4. Navigate to `/marketing/tools` with a site with Jetpack Professional and verify it matches the following screenshot:
<img width="503" alt="Screen Shot 2020-04-16 at 2 32 27 PM" src="https://user-images.githubusercontent.com/2810519/79510081-91e87d00-7ff1-11ea-8cda-c012c4f1fbfb.png">
6. Navigate to `/marketing/tools` with a site with a WordPress.com Premium plan or lower and verify it matches the following screenshot: 
<img width="491" alt="Screen Shot 2020-04-16 at 2 42 25 PM" src="https://user-images.githubusercontent.com/2810519/79510355-163b0000-7ff2-11ea-961d-56c22bbb4a3e.png">
7. Click "Upgrade to Business" and verify it takes you to `/plans` with the business plan "suggested"
<img width="200" alt="Screen Shot 2020-04-16 at 2 55 00 PM" src="https://user-images.githubusercontent.com/2810519/79510470-408cbd80-7ff2-11ea-8fef-d74ebdd7a32c.png">



Fixes #39603
